src/site/index.njk

```diff
@@ -1,8 +1,8 @@
 ---
 title: Home
 description: >-
-  MOD.UK Frontend contains the code for components you’ll need to build user
-  interfaces for Defence products and services.
+  Design and build consistent Defence services.
+headTitle:
```

Comment:
This is to stop `Home – ` being added to the title on the home page (was never intentional).